### PR TITLE
Update ready transition instructions

### PR DESCRIPTION
**Pull request by mtesseract**
## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Documentation added if necessary
- [ ] CI and all relevant tests are passing

## Test manual

TODO: Add manual testing efforts

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```

### CODE REVIEW

**Review by johannes94 — CHANGES_REQUESTED** [review]

File: dev/env/central-to-ready/TESTING-HOWTO.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ $ fleet-manager serve
 ## Test
 
 ```
-$ fmcurl rhacs/v1/agent-clusters/1234567890abcdef1234567890abcdef/dinosaurs
+$ fmcurl rhacs/v1/agent-clusters/1234567890abcdef1234567890abcdef/centrals
```
[the remark this answers is not in the document]
Comment (mtesseract) [reply dev/env/central-to-ready/TESTING-HOWTO.md:22]: Btw, the script has a normalizing step built-in, so it' doesnt't matter if you have a slash at the beginning or not.

File: dev/env/central-to-ready/TESTING-HOWTO.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ $ fleet-manager serve
 ## Test
```
[the remark this answers is not in the document]
Comment (mtesseract) [reply dev/env/central-to-ready/TESTING-HOWTO.md:20]: Good point, thanks

File: dev/env/central-to-ready/central-status-update-ready.json [hunk 3]
```diff
@@ -17,8 +17,8 @@
             }
         ],
         "versions": {
-            "dinosaur": "2.4.1",
-            "dinosaurOperator": "0.21.2"
+            "dinosaur": "0.1.0",
```
[the remark this answers is not in the document]
Comment (mtesseract) [reply dev/env/central-to-ready/central-status-update-ready.json:20]: Oh my. Thanks.

File: dev/env/central-to-ready/TESTING-HOWTO.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (johannes94) [reply dev/env/central-to-ready/TESTING-HOWTO.md:28]: Ok, I have it sourced through my .zshrc as a bash function. Also probably my version of the script is different does not normalize the slashes. I'm fine if it stays like this.

**Review by johannes94 — APPROVED** [review]

**Pull request merged**